knife/Gemfile

```diff
@@ -24,3 +24,7 @@ gem "ohai", git: "https://github.com/chef/ohai.git", branch: "main"
 gem "chef", path: ".."
 gem "chef-utils", path: File.expand_path("../chef-utils", __dir__) if File.exist?(File.expand_path("../chef-utils", __dir__))
 gem "chef-config", path: File.expand_path("../chef-config", __dir__) if File.exist?(File.expand_path("../chef-config", __dir__))
+
+source "https://artifactory-internal.ps.chef.co/artifactory/api/gems/omnibus-gems-local/" do
+  gem "chef-licensing"
+end
```

Comment:
I'm assuming this is just temporary?

Reply:
Yeah right this is temporary as of now for testing. Once the chef-licensing gem is published we will remove this and will be fetching this gem from knife.gemspec